.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+
+jobs:
+   # TODO: Uncomment pod-lint once we bump Cococapods.
+   # We need a release with https://github.com/CocoaPods/CocoaPods/pull/11660
+  # pod-lint:
+  #   name: Pod Lint
+  #   runs-on: macOS-12
+  #   steps:
+  #     - uses: actions/setup-ruby@v1
+  #       with:
+  #         ruby-version: '2.7.6'
+  #     - name: Checkout Repo
+  #       uses: actions/checkout@v3
+  #     - name: Bundle Install
+  #       run: bundle install
+  #     - name: Select Xcode Version
+  #       run: sudo xcode-select --switch /Applications/Xcode_14.1.0.app/Contents/Developer
+  #     - name: Lint Podspec
+  #       run: bundle exec pod lib lint --verbose --fail-fast --swift-version=5.7
+  spm-14:
+    name: Build Xcode 14
+    runs-on: macOS-12
+    strategy:
+      matrix:
+        platforms: [
+          'iOS_16,watchOS_9',
+          'macOS_13,tvOS_16',
+        ]
+      fail-fast: false
+    steps:
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.7.6'
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+      - name: Bundle Install
+        run: bundle install
+      - name: Select Xcode Version
+        run: sudo xcode-select --switch /Applications/Xcode_14.1.0.app/Contents/Developer
+      - name: Build and Test Framework
+        run: Scripts/build.swift ${{ matrix.platforms }}
+      - name: Prepare Coverage Reports
+        run: ./Scripts/prepare-coverage-reports.sh
+      - name: Upload Coverage Reports
+        if: success()
+        uses: codecov/codecov-action@v3
```

Comment:
I'm surprised we're not yet getting any code coverage posts just yet, but we also have nearly zero code in the repo. I may put in a pin in this and hope it gets fixed by adding code or running this code on `main`. I'll fix this if either of the prior do not work.

Reply:
It's working! https://github.com/dfed/swift-async-queue/pull/2#issuecomment-1324577222